image_generation/stable_diffusion_xl/main.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from pathlib import Path
+from openvino.runtime import Core
+from optimum.intel.openvino import OVStableDiffusionXLPipeline
+
+#Specify model dir
+model_dir = Path("openvino-sd-xl-base-1.0")
+#model_dir = Path("openvino-ssd-1b") #ssd-1b
+#model_dir = Path("openvino-sdxl-turbo") #sdxl-turbo
+
+#Define model parameters
+core = Core()
+device = "GPU"
+text_prompt = "valley in the Alps at sunset, epic vista, beautiful landscape, 4k, 8k"
+neg_prompt = None
+#neg_prompt = "ugly, blurry, poor quality" #Negative prompt for SSD-B1 model
+num_inference_steps = 25
+seed = 42
+
+#Define and load the model
+text2image_pipe = OVStableDiffusionXLPipeline.from_pretrained(model_dir, device=device)
```

Comment:
@slyalin, is one of the goals to directly including the Optimum-Intel pipeline implementations in the repo (e.g., [OVStableDiffusionXLPipeline ](https://github.com/huggingface/optimum-intel/blob/f32d5015de83ec561e8e4b7d3b19fdb8ecd6595f/optimum/intel/openvino/modeling_diffusion.py#L833) in the repo? Thank you!

Reply:
the goal is to avoid dependency on HF and implement missing parts (scheduler in this case) from scratch

Reply:
Thank you, @ilya-lavrenov! I can look into porting the [LMSDiscreteScheduler ](https://github.com/openvinotoolkit/openvino.genai/blob/4b0a2edd709a61b85cc407be2ea79de594dc5925/image_generation/stable_diffusion_1_5/cpp/src/stable_diffusion.hpp#L75C20-L75C40) to Python.

I noticed in another places of the repo, [the scheduler is also saved](https://github.com/openvinotoolkit/openvino.genai/blob/4b0a2edd709a61b85cc407be2ea79de594dc5925/llm_bench/python/convert.py#L358) - is this another approach to consider?